### ISSUE TITLE
Typo fix

### ISSUE THREAD

**Issue by maddyrombes** [jfangrad/react-dropdown-aria#18]
There is a typo on line 92 of Dropdown.tsx: `ari-describedby` should be `aria-describedby`.
